test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
```

Comment:
Cool. I don't doubt you changed the name, but I'm also not seeing a test called "a custom token function provided to .token(name,fn)"? What line is that on?